ros2_ws/VERSION.jazzy

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
might be nice if we can use pre-release tags like `2.0.0-jazzy.rc.1` to leave room for ordered release candidates without abusing the 0.0.1 tag. I haven't checked deeply how that would propagate with the CI and following package / app builders (@LouisBrunner)

Reply:
Currently, the CI couldn't handle this

Reply:
ok then let's go with the 0.0.1 approach for now